charms/precise/clearwater-bono/README.md

```diff
@@ -1,88 +1,50 @@
 # Overview
 
-This charm supports deployment and scaling of the Bono component of a Project Clearwater system.  See http://www.projectclearwater.org for more information on Project Clearwater.
-
-# Usage
-
-The Bono service should be deployed as part of a Clearwater system.  A Clearwater system can be deployed in a Juju environment by creating a config.yaml file then running the following commands.
-
-    juju deploy --config config.yaml clearwater-route53
-    juju deploy --config config.yaml --constraints arch=amd64 clearwater-ellis
-    juju deploy --config config.yaml --constraints arch=amd64 clearwater-bono
-    juju deploy --config config.yaml --constraints arch=amd64 clearwater-sprout
-    juju deploy --config config.yaml --constraints arch=amd64 clearwater-homestead
-    juju deploy --config config.yaml --constraints arch=amd64 clearwater-homer
-    juju add-relation clearwater-ellis clearwater-route53:register-ellis
-    juju add-relation clearwater-bono clearwater-route53:register-bono
-    juju add-relation clearwater-sprout clearwater-route53:register-sprout
-    juju add-relation clearwater-homestead clearwater-route53:register-homestead
-    juju add-relation clearwater-homer clearwater-route53:register-homer
-    juju expose clearwater-bono
-    juju expose clearwater-ellis
-
-The config.yaml configuration file takes the following format.
-
-    clearwater-route53:
-      zone: <domain>
-      access_key: <ec2 access key>
-      secret_key: <ec2 secret access key>
-      sas: "0.0.0.0"
-
-    clearwater-ellis:
-      smtp_server: smtp.cw-ngv.com
-      smtp_username: username
-      smtp_password: password
-      email_sender: blackhole@cw-ngv.com<mailto:blackhole@cw-ngv.com>
-      signup_key: secret
-      base_number: "6505550000"
-      number_count: 1000
-      repo: http://repo.cw-ngv.com/stable
-
-    clearwater-bono:
-      turn_workaround: password
-      repo: http://repo.cw-ngv.com/stable
-
-    clearwater-sprout:
-      reg_min_expires: 400
-      session_max_expires: 900
-      repo: http://repo.cw-ngv.com/stable
-
-    clearwater-homestead:
-      repo: http://repo.cw-ngv.com/stable
-
-    clearwater-homer:
-      repo: http://repo.cw-ngv.com/stable
-
-Note that the clearwater-bono charm can only be deployed on the `amd64` architecture.
+This is a [Juju charm](https://jujucharms.com/about), which allows deployment and scaling of the Bono component of a [Project Clearwater](http://projectclearwater.org) IMS core.
```

Comment:
I'd like to see a link to the Sprout repository/Bono documentation somewhere in the readme.